app/src/main/java/com/insalyon/les24heures/model/LiveUpdate.java

```diff
@@ -0,0 +1,54 @@
+package com.insalyon.les24heures.model;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+/**
+ * Created by lbillon on 4/22/15.
+ */
+public class LiveUpdate implements Parcelable {
+
+    private String title;
+    private String message;
+    private long timePublished;
+
+
```

Comment:
Tu as besoin d'un constructeur LiveUpdate(Parcel in, int flags) pour reconstruire l'object.

(si ca marche sans, je suis preneur de le savoir)